mobile/lib/shared/providers/asset.provider.dart

```diff
@@ -377,16 +384,27 @@ QueryBuilder<Asset, Asset, QAfterSortBy>? getRemoteAssetQuery(WidgetRef ref) {
       .sortByFileCreatedAtDesc();
 }
 
-IsarCollection<Asset> _assets(StreamProviderRef<RenderList> ref) =>
+IsarCollection<Asset> _assets(AutoDisposeStreamProviderRef<RenderList> ref) =>
     ref.watch(dbProvider).assets;
 
 QueryBuilder<Asset, Asset, QAfterSortBy> _commonFilterAndSort(
+  AutoDisposeStreamProviderRef<RenderList> ref,
   QueryBuilder<Asset, Asset, QAfterWhereClause> query,
 ) {
+  final localIds =
+      ref.watch(deviceAssetsProvider).valueOrNull?.assetIdsForBackup ??
+          <String>[];
+
   return query
       .filter()
       .isArchivedEqualTo(false)
       .isTrashedEqualTo(false)
       .stackParentIdIsNull()
+      .group(
```

Comment:
What's the purpose of group operation here?

Reply:
It is to make the timeline related filter for local assets more obvious.